src/components/sidebar/sidebar.component.js

```diff
@@ -82,6 +85,8 @@ Sidebar.propTypes = {
   position: PropTypes.string,
   /** Sets the size of the sidebar when open. */
   size: PropTypes.string,
+  /** Node that will be used as sidebar header. */
+  header: PropTypes.node,
```

Comment:
We need to add the styled-system padding props to the propTypes here as well and make sure they are passed down to `SidebarStyle`